target/linux/ramips/image/mt7620.mk

```diff
@@ -569,6 +569,15 @@ define Device/lava_lr-25g001
 endef
 TARGET_DEVICES += lava_lr-25g001
 
+define Device/lb-link_bl-w1200
+  SOC := mt7620a
+  DEVICE_VENDOR := LB-Link
+  DEVICE_MODEL := BL-W1200
+  IMAGE_SIZE := 7872k
+  DEVICE_PACKAGES := kmod-usb2 kmod-usb-ohci kmod-mt76x2 kmod-phy-realtek
```

Comment:
What is "kmod-phy-realtek" used for? Is it related to "mediatek,port4"?

Reply:
My bad. It shouldn't be here.